nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RHostedRecipe.groovy

```diff
@@ -114,6 +122,11 @@ class RHostedRecipe
         .handler(hostedHandlers.putArchive)
         .create())
 
+    builder.route(metadataMatcher()
```

Comment:
Add a matcher for unsupported metadata files

Reply:
Done in 0773e4b6c35b56f38b1ee7b49cd33eeb3a4603cb